src/api/app/controllers/webui/package_controller.rb

```diff
@@ -558,7 +558,14 @@ def branch
                                 user: User.current.login)
 
     flash[:notice] = 'Successfully branched package'
-    redirect_to(package_show_path(project: created_project_name, package: created_package_name))
+
+    branched_package_object = Package.find_by_project_and_name(created_project_name, created_package_name)
+
+    if request.env["HTTP_REFERER"] == image_templates_url && branched_package_object.kiwi_image?
+      redirect_to(import_kiwi_image_path(branched_package_object.id))
```

Comment:
Hmm, maybe we can change the route to also accept project and package name?
https://github.com/openSUSE/open-build-service/blob/master/src/api/config/routes.rb#L195
(I'm fine with doing it in a dedicated PR)

Reply:
Better in another PR 👍